src/components/EntityInfo/EntityMenu.vue

```diff
@@ -219,6 +219,7 @@
               >
                 <template #activator="{ on }">
                   <div
+                    v-if="enableAmalgamation"
```

Comment:
Are you sure we don't want to show it at all right now?

Reply:
Ask Mihai.

Reply:
Looks like we're going with remove Sev:
![remove](https://github.com/bcgov/business-filings-ui/assets/122301442/a5cdcbaf-455d-42aa-8e17-cf98d072847c)

Reply:
OK.

I would tease the users. (They could learn what's coming and where it will be.) But not my call.

Reply:
Funny that comment came from Yui. In the past, he's advocated for "teaching" users where things are / will be.

Reply:
> I would tease the users. (They could learn what's coming and where it will be.) But not my call.

I agree.